src/hotspot/share/opto/escape.cpp

```diff
@@ -1264,7 +1278,24 @@ bool ConnectionGraph::reduce_phi_on_safepoints_helper(Node* ophi, Node* cast, No
   return true;
 }
 
-void ConnectionGraph::reduce_phi(PhiNode* ophi, GrowableArray<Node *>  &alloc_worklist, GrowableArray<Node *>  &memnode_worklist) {
+void ConnectionGraph::reduce_phi(PhiNode* ophi, GrowableArray<Node *>  &alloc_worklist, GrowableArray<Node *>  &memnode_worklist, Unique_Node_List &reducible_merges) {
+  Unique_Node_List nested_phis;
+  // Collect nested phi nodes
```

Comment:
Let's decide on a way to name nested phis and stick with it (in some places we use 'nestedphis', all one word). 1 or 2 words? Any capitalization?